src/markerclusterer.js

```diff
@@ -1672,3 +1672,5 @@ if (typeof String.prototype.trim !== 'function') {
   };
 }
 
+module.exports = MarkerClusterer;
```

Comment:
What is this supposed to do when module isn't defined? Should you not opt to do the UMD style if you want this to continue working for other people as well:

Article about UMD (with links to decent Github examples, as well): http://bob.yexley.net/umd-javascript-that-runs-anywhere/